research.html

```diff
@@ -0,0 +1,61 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="utf-8">
+  <meta name="Description" content="INF 7420 Project">
+  <meta name="viewport" content="width=device-width, initial-scale=1">
+  <title>Dr. K.P. Kohn - Research</title>
+</head>
+<body>
+<header>
+  <nav>
+    <ul>
+      <li>
+        <a href="index.html">Home</a>
+      </li>
+      <li>
+        <a href="about.html">About</a>
+      </li>
+      <li>
+        <a href="research.html">Research</a>
+     </li>
+     </ul>
+    </nav>
+</header>
+<main>
+<h1>Connecting Chemistry and Biology: Exploring Students' Perceptions of College Courses</h1>
+<p>There have been few research studies that explore undergraduate students' understanding in different discipinary contexts. Instead,many have theorized the benefits of integrated learning experiences that belnd and synthesize different discipinary knowledge and perspectives. Thse that have been described in the literature vary widely in their implementation and often provide little in the way of evidence on student learning gains. The <abbr title="Discipline-Based Education Research">DBER</abbr> report confirmed this gap in the research literature, recomming that interdisciplinary studies on crosscutting concepts be conducted.
```

Comment:
Break the paragraphs up. End the first paragraph on line 27 and start a new one on line 29.